tests/apps/tests.py

```diff
@@ -45,9 +45,12 @@ def test_ready(self):
         Tests the ready property of the master registry.
         """
         # The master app registry is always ready when the tests run.
-        self.assertIs(apps.ready, True)
+        self.assertTrue(apps.ready)
```

Comment:
This change doesn't look necessary. We prefer assertIs in most cases because assertTrue passes for bool(value) which might not be what we want.